index.js

```diff
@@ -19,12 +19,15 @@ const linkify = (href, options) => createHtmlElement({
 // Get DOM node from HTML
 const domify = html => document.createRange().createContextualFragment(html);
 
-const getAsString = (string, options) => string.replace(urlRegex(), match => linkify(match, options));
+const isTruncated = (index, entries) => index + 2 === entries.length && (entries[entries.length - 1][1] === '…' || (entries[index][1].endsWith('...') && entries[entries.length - 1][1] === ''));
```

Comment:
```suggestion
const isTruncated = (index, entries) => index + 2 === entries.length && (entries[entries.length - 1][1] === '…' || (entries[index][1].endsWith('…') && entries[entries.length - 1][1] === ''));
```

Reply:
Intentional, see https://github.com/sindresorhus/linkify-urls/pull/40#discussion_r1304299859

Reply:
No this was my suggestion to support both. 

Also this long condition is hard to follow. It should probably be broken up into "if else and return" plus comments.

Reply:
[`f2e5d4c` (#40)](https://github.com/sindresorhus/linkify-urls/pull/40/commits/f2e5d4c0c1b92bc112762eb1f7d5643c04adaeba)